docs/guides/getting-started.md

```diff
@@ -73,8 +73,13 @@ As seen above, connecting to an organization can be done by calling the `connect
 
 It requires two parameters:
 
+<<<<<<< HEAD
```

Comment:
This file still has a few merge conflicts.

Reply:
oops :see_no_evil: